packages/core/package.json

```diff
@@ -13,11 +13,12 @@
     "directory": "packages/core"
   },
   "scripts": {
+    "clean": "rimraf *.tsbuildinfo dist",
     "lint": "eslint '{src,bin,test}/**/*.ts'",
     "test": "mocha -r ts-node/register 'test/**/*.test.ts'",
     "mocha": "mocha -r ts-node/register",
     "build": "yarn prepack",
-    "prepack": "copy \"package.json\" \"dist\"  && tsc --build tsconfig.esm.json && tsc --build tsconfig.cjs.json"
+    "prepack": "yarn clean && tsc --build --clean && copy \"package.json\" \"dist\"  && tsc --build tsconfig.esm.json && tsc --build tsconfig.cjs.json"
```

Comment:
Why is this necessary? Won't it circumvent the optimizations we have by using references?

Reply:
It was just something I was trying in desperation to get this passing on windows.  It is removed now